src/components/NglViewer/NglViewer.css

```
@@ -0,0 +1,20 @@
+.viewport {
+  width: 400px;
+  height: 400px;
```

The structure viewer is a bit small. It does allow you to see the table behind it, but I would like it to be a bigger.
```suggestion
  width: 800px;
  height: 600px;
```